src/app/components/MediaLoader/index.tsx

```diff
@@ -298,6 +291,7 @@ const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
         )}
         {captionBlock && (
           <Caption
+            className={orientation === 'portrait' ? 'portrait-caption' : ''}
```

Comment:
Do we want to extract this into another variable to improve readability (and update all references)? 

```suggestion
            const isPortraitVideo = orientation === 'portrait';
            ...
            className={isPortraitVideo ? 'portrait-caption' : ''}
```